src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String FIRST_LINE_OF_INPUT_FILE = "fruit,quantity";
+    private static final String DELIMITER = ",";
+
+    @Override
+    public String getReport() {
+        Map<String, Integer> sortedFruitsMap = new TreeMap<>(Storage.remainsOfFruits);
+        StringBuilder stringBuilder = new StringBuilder(FIRST_LINE_OF_INPUT_FILE);
```

Comment:
don't name collections/containers like this (stringBuilder), name them after what they store